slice/Ice/Version.ice

```diff
@@ -33,11 +33,15 @@ struct ProtocolVersion
     byte minor;
 }
 
+#ifndef __SLICE2CPP__
+// Ice for C++ provides an hand-written equivalent.
+
 /// A version structure for the encoding version.
 struct EncodingVersion
```

Comment:
It wouldn't work. Fortunately no other Slice definition references EncodingVersion.

As a result, should remove EncodingVersion from Slice rather than ifdef-it-out for C++?

Reply:
Yes, I think we should